metagpt/utils/mermaid.py

```diff
@@ -126,6 +121,46 @@ class KnowledgeBase {
     Index --> KnowledgeBase
 """
 
+MMC1_REFINE = """
+classDiagram
+    class Main {
+        -SearchEngine search_engine
+        +main() str
+        +newMethod() str  # Incremental change
+    }
+    class SearchEngine {
+        -Index index
+        -Ranking ranking
+        -Summary summary
+        +search(query: str) str
+        +newMethod() str  # Incremental change
```

Comment:
In Mermaid, this form of annotation is illegal. This will cause the output to not be rendered